specification.md

```diff
@@ -3146,6 +3143,17 @@ the needed events at the defined times to trigger workflow instance creation.
 
 #### Schedule Definition
 
+`Schedule` definition can have two types, either `string` or `object`.
+If `string` type, it defines time interval describing when the workflow instance can be created.
```

Comment:
I don't really understand why this 'interval' property exists and why it's related to the 'schedule'.  I can see 2 potential use-cases:
a) The ability to define a 'validity' period for a workflow, which limits the time period within which it's allowed to be launched.  Outside of that time period, the runtime would reject launch requests for this workflow.
b) The ability to define an 'interval' for periodic launches instead of cron.  For example, run the workflow every 7 minutes, or perhaps run it 7 minutes after the last time it completed.

For use-case A, it seems like we should elevate this parameter outside of the 'schedule' since it seems more generally useful than just for limiting schedules.  My personal opinion is that this feature isn't necessary, since it doesn't feel super common and a workflow could enforce this itself by checking the time period in it's first step and ending early.

For use-case B, I could see customers wanting something other than cron but I'm not sure if that has been a requested feature or not yet.

Reply:
You can write a schedule w/ it imo, for example:

R/0402T/P1Y is a repeating interval of "repeat every year on april second"

or

R/20210401/P1M is repeating interval of "repeat every month on first of month starting with april 1st"

is that not a schedule?

Reply:
Got ya, I was confused because all the examples are shown as a time range, which it's hard to understand how that works as a schedule.